spec/person_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../person'
+require_relative '../corrector'
+
+describe Person do
+  before(:each) do
```

Comment:
Please add tests for all public methods of the `Person` class.

Reply:
Thank you. We added the tests for all Public methods as you requested